install/GNUmakefile

```diff
@@ -0,0 +1,212 @@
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021 Intel Corporation
+
+# The default target of this installation Makefile copies firmware and
+# topology files produced by ./scripts/xtensa-build-all.sh into a tree
+# structure (staging area) identical to /lib/firmware/intel/{sof,sof-tplg}/
+#
+# After inspecting the staging area, you can run "[sudo] make rsync" to
+# rsync it into either a local or remote /lib/firmware/
+# directory, or to a sof-bin clone or to any other destination of your
+# choice configured in the config.mk file.
+
+
+.DEFAULT_GOAL := stage
+.PHONY: clean stage rsync
+.PHONY: signed unsigned ldicts topologies
+.PHONY: compare signed_dummies
+
+# Local configuration
+-include config.mk
+
+
+       ###    platforms   ###
+
+# To build only a couple, override these in config.mk
+
+UNSIGNED_list ?= bdw byt cht
+SIGNED_list  ?= apl cnl icl jsl tgl
+# older SOF versions
+# SIGNED_list += hsw kbl skl sue
+ALIAS_list ?= glk cfl cml ehl
+
+$(info UNSIGNED_list = ${UNSIGNED_list} )
+$(info SIGNED_list   = ${SIGNED_list}   )
+$(info ALIAS_list    = ${ALIAS_list}    )
+
+target_of_glk := apl
+target_of_cfl := cnl
+target_of_cml := cnl
+target_of_ehl := tgl
+
+
+# Try to keep SOF_VERSION optional if possible
+
+SOF_VERSION ?= $(shell git describe --dirty)
+ifneq (${SOF_VERSION},)
+VERSION_DIR := ${SOF_VERSION}/
+VERSION_SUFFIX := -${SOF_VERSION}
+endif
+
+TOOLCHAIN ?= gcc
```

Comment:
for release binary, we will always build with xcc from CI. So a local build scripts is not needed here.